### DISCUSSION TITLE
object column veh_dict_id mush in insert into columns

### DISCUSSION THREAD

**Discussion by wentaotang0** [apache/doris#21187] [RFC]
//show create table
CREATE TABLE `t_task_pushed_user` (
  `task_id` bigint(20) NOT NULL COMMENT "任务id",
  `task_start_time` datetime NOT NULL COMMENT "任务开始时间",
  `user_uid` bitmap BITMAP_UNION DEFAULT NULL COMMENT "用户id",
  `veh_dict_id` bitmap BITMAP_UNION DEFAULT NULL COMMENT "指令下发唯一id"
) ENGINE=OLAP
AGGREGATE KEY(`task_id`, `task_start_time`)
COMMENT "任务已推送用户表"
DISTRIBUTED BY HASH(`task_id`, `task_start_time`) BUCKETS 4
PROPERTIES (
"replication_allocation" = "tag.location.default: 2",
"in_memory" = "false",
"storage_format" = "V2"
);

//insert
insert into t_task_pushed_user (task_id,task_start_time,user_uid) values (530,'2023-06-26 16:18:00',to_bitmap(101113));

//question
1105 - errCode = 2, detailMessage =  object column veh_dict_id mush in insert into columns, Time: 0.012000s

`veh_dict_id` bitmap BITMAP_UNION DEFAULT NULL，why this happen?

**Comment by bingquanzhao** [comment]
Currently, object-stored column columns cannot be overwritten with default values. You can attempt to insert data explicitly.

insert into t_task_pushed_user (task_id,task_start_time,user_uid,veh_dict_id) values (530,'2023-06-26 16:18:00',to_bitmap(101113),to_bitmap(null));
